Rakefile

```diff
@@ -0,0 +1,19 @@
+task :generate do
+  require 'open-uri'
+  require 'fileutils'
+  content = open("https://raw.githubusercontent.com/fastlane/fastlane/master/Rakefile").read
+  content = content.split("desc").first # only the first lines are relevant
+  eval(content)
```

Comment:
Agreed. What do you suggest? Should we maintain yet another list of tools? Or is it time to have a REST API for a list of all tools, so we can access it from our back-end services and tools?